contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -326,7 +326,23 @@ contract WithdrawManager is WithdrawManagerStorage /* , IWithdrawManager */ {
           delete ownerExits[keccak256(abi.encodePacked(_token, currentExit.owner))];
         }
 
-        IDepositManager(registry.getDepositManagerAddress()).transferAmount(_token, exitOwner, currentExit.receiptAmountOrNFTId);
+        address depositManager = registry.getDepositManagerAddress(); // TODO: make assembly call and reuse memPtr
+        uint256 amount = currentExit.receiptAmountOrNFTId;
+        uint256 _gas = gasLimit; // can't read gloablvars in asm
+        assembly {
+          let ptr := mload(64)
+          // keccak256('transferAmount(address,address,uint256)') & 0xFFFFFFFF00000000000000000000000000000000000000000000000000000000
+          mstore(ptr, 0x01f4747100000000000000000000000000000000000000000000000000000000)
+          mstore(add(ptr, 4), and(_token, 0xffffffffffffffffffffffffffffffffffffffff)) // zero-out the unused bytes
+          mstore(add(ptr, 36), and(exitOwner,0xffffffffffffffffffffffffffffffffffffffff))
+          mstore(add(ptr, 68), amount) // TODO: read directly from struct
+
+          let result := call(_gas, depositManager, 0, ptr, 100, ptr, 32)
+
+          if eq(result, 0) { // if false revert
```

Comment:
1. Verify that `result` is 0 on revert - solidity docs don't specifically mention (they just talk about outOfGas)
2. Check both output area `(ptr, 32)` and return value `result` to be truthy.

Reply:
```
assembly {
            let ptr := mload(0x40)
            // keccak256('transferAmount(address,address,uint256)') & 0xFFFFFFFF00000000000000000000000000000000000000000000000000000000
            mstore(ptr, 0x01f4747100000000000000000000000000000000000000000000000000000000)
            mstore(add(ptr, 4), add(_token,0))
            mstore(add(ptr, 36), add(_token1,0))
            mstore(add(ptr, 68), add(amount,0))
            
            let ans := add(ptr, 100)
            let result := call(gas, depositManager, 0, ptr, 100, ans, 32)
            if eq(and(result, mload(ans)), 0) {
                revert(0,0)
            }
}
```